content/get-started/multisig-accounts.mdx

```diff
@@ -0,0 +1,201 @@
+----------------
+ifmsaccountAKLQe4HMT9QpBO9Kls3NHIstM05KO01NUFZb3RicZCwBAAAAgq3GMK8peqPmGFW+q/xyYhHsVAE1PUkSpHKu+gX7kVi5CgAA7sDEk6AaayQmo89XY4WMS...
+  `}
+/>
+
+The command above outputs a list of serialized accounts, one per participant. The account for a participant contains the shared group keys and the signing key for that participant. Each participant's account is encrypted, so only the participant who created the matching identity can import the account.
+
+The dealer should share the output of the command with the multisig participants and each participant should import their account:
+
+```sh
+ironfish wallet:import
+```
+
+## Create and sign a transaction with multisig
+
+Sending a transaction from a multisig account requires that at least the minimum number of signers (set at account creation) signs the transaction.
+
+### Step 1: Create an unsigned transaction
+
+Any participant in the group can initiate sending a transaction by creating an unsigned transaction using the `send` command:
+
+```sh
+ironfish wallet:send --unsignedTransaction
+```
+
+After creating the unsigned transaction the participant must share it with the rest of the group.
+
+### Step 2: Identify signers
+
+Before beginning the signing process, participants who will sign the transaction must communicate that they plan to sign the transaction and share their identity with other signers. This list of identities for planned signers is an input to the next step.
+
+### Step 3: Create signing commitments
+
+Each member of the group must first create a "signing commitment" for the given transaction and list of signer identities.
+
+```sh
+ironfish wallet:multisig:commitment:create
+```
+
+Example:
+<Terminal command="ironfish wallet:multisig:commitment:create \
+-i 723f415e10ec47955036105834ab7dcc...89557882a9d3834e9ee9a82dc4ac2f0a \
+-i 721cb9582cec837588df0b08d77d4870...2b269850be9d80d1f8360fa57df0a306 \
+-u 01010000000000000002000000000000...9f3fb432907a84c8483586a6a565d308"
+  output={`
+==================
+Notes sent:
+==================
+
+Amount:        $IRON 0.00000010
+Memo:          
+Recipient:     95b4d767410460fb8f093c1261c9a499ceae23801c14bde88675b27cf1c5718c
+Sender:        ca860e5c6c87118f738e1764a87b1cec1913e3e624237bddecb88d289a445c2c
+
+
+==================
+Notes received:
+==================
+
+Amount:        $IRON 0.00099989
+Memo:          
+Recipient:     ca860e5c6c87118f738e1764a87b1cec1913e3e624237bddecb88d289a445c2c
+Sender:        ca860e5c6c87118f738e1764a87b1cec1913e3e624237bddecb88d289a445c2c
+
+
+Confirm signing commitment creation (Y/N): Y
+
+Commitment:
+
+07749d787c6f3c0bce91fb3f3869c96b...342022941ff351bd60723b9a9aa2d29a
+  `}
+/>
+
+Each participant must use the same list of identities from Step 2.
+
+### Step 4: Aggregate signing commitments
+
+One participant must act as a "coordinator" and aggregate all signing commitments from Step 3 to produce the "signing package" for the transaction.
```

Comment:
Like above: this could be a view-only account; it doesn't need to be a participant